app/views/hub/outgoing_emails/unsubscribe_email.html.erb

```diff
@@ -0,0 +1,17 @@
+<% content_for :page_title, t(".title") %>
```

Comment:
Nitpicky but also I think the naming of this page confuses me, can we call it "unsubscribe_from_emails_page` or something. I think when I see `unsubscribe_email` I think its an mailer